domains/baptiste.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "BapRx",
+        "email": "rouxbaptiste@outlook.com"
+    },
+    "record": {
+        "CNAME": "baprx.github.io"
```

Comment:
I cannot reach your website (404 page)

Reply:
I'm currently working on it, shall I wait for it to be ready?